base/mmcif_ma-core.dic

```diff
@@ -8026,6 +8048,16 @@ save__ma_experimental_validation.reference_type
      'other'
      save_
 
+save__ma_experimental_validation.reference_doi
+    _item_description.description
+;    The Digital Object Identifier (DOI) for the experimental validation information. 
```

Comment:
To make it explicit, let's add a "If available, this should be provided instead of reference_url."